src/client/datascience/editor-integration/codewatcher.ts

```diff
@@ -977,34 +990,14 @@ export class CodeWatcher implements ICodeWatcher {
         }
     }
 
-    private async addCode(
-        code: string,
-        file: Uri,
-        line: number,
-        leftCount: number,
-        editor?: TextEditor,
-        debug?: boolean
-    ): Promise<boolean> {
-        this.addCodePromise = this.addCodePromise.then((_previousResult) => {
```

Comment:
The promise chain wasn't necessary here anymore because the IW handles queueing itself now.

Reply:
Per this comment why is chainable needed here then?